src/Metadata/Extractor/YamlExtractor.php

```diff
@@ -32,7 +32,7 @@
     protected function extractPath(string $path)
     {
         try {
-            $resourcesYaml = Yaml::parse(file_get_contents($path));
+            $resourcesYaml = Yaml::parse(file_get_contents($path), Yaml::PARSE_KEYS_AS_STRINGS);
```

Comment:
> IIRC, using this constant is deprecated

Are you sure? I can't find any information on this subject and I don't see anything in the code of the component.